WoWsShipBuilder.Core/DataContainers/Armament/SecondaryBatteryDataContainer.cs

```diff
@@ -8,6 +8,7 @@
 using WoWsShipBuilder.DataElements.DataElementAttributes;
 using WoWsShipBuilder.DataElements.DataElements;
 using WoWsShipBuilder.DataStructures;
+using WoWsShipBuilder.DataStructures.Ship;
 
 namespace WoWsShipBuilder.Core.DataContainers
```

Comment:
Dangling TODO

Reply:
Again, Todo is not from me but from itto. No idea what issue there was